qa/rpc-tests/mempool_reorg.py

```diff
@@ -67,8 +69,8 @@ def run_test(self):
         assert_raises(JSONRPCException, self.nodes[0].sendrawtransaction, timelock_tx)
 
         # Create 102_1 and 103_1:
-        spend_102_1_raw = create_tx(self.nodes[0], spend_102_id, node1_address, 49.98)
-        spend_103_1_raw = create_tx(self.nodes[0], spend_103_id, node1_address, 49.98)
+        spend_102_1_raw = create_tx(self.nodes[0], spend_102_id, node1_address, 0.1)
+        spend_103_1_raw = create_tx(self.nodes[0], spend_103_id, node1_address, 1)
```

Comment:
weird, Why these two lines pass? It should fail because input is not enough to pay the output and fee.